utils/build/rollup.config.js

```diff
@@ -263,6 +263,21 @@ ${ code }`;
 
 }
 
+function deprecationWarning() {
+
+	return {
+
+		renderChunk( code ) {
+
+			return `console.warn( 'This build file will be removed with r160. Have a look at the Installation guide for alternative three.js setups: https://threejs.org/docs/index.html#manual/en/introduction/Installation' );
```

Comment:
It may be unclear what "this build file" is, or even whether it's coming from three.js if the file has been bundled in with other JavaScript. Suggested wording:

> Scripts 'build/three.js' and 'build/three.min.js' are deprecated with r150+, and will be removed with r160. Please use ES Modules or alternatives:
> 
> https://threejs.org/docs/index.html#manual/en/introduction/Installation